wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php

```diff
@@ -0,0 +1,1000 @@
+		WP_MySQL_Lexer::INT_SYMBOL                => 'int',
+		WP_MySQL_Lexer::TINYINT_SYMBOL            => 'tinyint',
+		WP_MySQL_Lexer::SMALLINT_SYMBOL           => 'smallint',
+		WP_MySQL_Lexer::MEDIUMINT_SYMBOL          => 'mediumint',
+		WP_MySQL_Lexer::BIGINT_SYMBOL             => 'bigint',
+		WP_MySQL_Lexer::REAL_SYMBOL               => 'double',
+		WP_MySQL_Lexer::DOUBLE_SYMBOL             => 'double',
+		WP_MySQL_Lexer::FLOAT_SYMBOL              => 'float',
+		WP_MySQL_Lexer::DECIMAL_SYMBOL            => 'decimal',
+		WP_MySQL_Lexer::NUMERIC_SYMBOL            => 'decimal',
+		WP_MySQL_Lexer::FIXED_SYMBOL              => 'decimal',
+		WP_MySQL_Lexer::BIT_SYMBOL                => 'bit',
+		WP_MySQL_Lexer::BOOL_SYMBOL               => 'tinyint',
+		WP_MySQL_Lexer::BOOLEAN_SYMBOL            => 'tinyint',
+		WP_MySQL_Lexer::BINARY_SYMBOL             => 'binary',
+		WP_MySQL_Lexer::VARBINARY_SYMBOL          => 'varbinary',
+		WP_MySQL_Lexer::YEAR_SYMBOL               => 'year',
+		WP_MySQL_Lexer::DATE_SYMBOL               => 'date',
+		WP_MySQL_Lexer::TIME_SYMBOL               => 'time',
+		WP_MySQL_Lexer::TIMESTAMP_SYMBOL          => 'timestamp',
+		WP_MySQL_Lexer::DATETIME_SYMBOL           => 'datetime',
+		WP_MySQL_Lexer::TINYBLOB_SYMBOL           => 'tinyblob',
+		WP_MySQL_Lexer::BLOB_SYMBOL               => 'blob',
+		WP_MySQL_Lexer::MEDIUMBLOB_SYMBOL         => 'mediumblob',
+		WP_MySQL_Lexer::LONGBLOB_SYMBOL           => 'longblob',
+		WP_MySQL_Lexer::TINYTEXT_SYMBOL           => 'tinytext',
+		WP_MySQL_Lexer::TEXT_SYMBOL               => 'text',
+		WP_MySQL_Lexer::MEDIUMTEXT_SYMBOL         => 'mediumtext',
+		WP_MySQL_Lexer::LONGTEXT_SYMBOL           => 'longtext',
+		WP_MySQL_Lexer::ENUM_SYMBOL               => 'enum',
+		WP_MySQL_Lexer::SET_SYMBOL                => 'set',
+		WP_MySQL_Lexer::SERIAL_SYMBOL             => 'bigint',
+		WP_MySQL_Lexer::GEOMETRY_SYMBOL           => 'geometry',
+		WP_MySQL_Lexer::GEOMETRYCOLLECTION_SYMBOL => 'geometrycollection',
+		WP_MySQL_Lexer::POINT_SYMBOL              => 'point',
+		WP_MySQL_Lexer::MULTIPOINT_SYMBOL         => 'multipoint',
+		WP_MySQL_Lexer::LINESTRING_SYMBOL         => 'linestring',
+		WP_MySQL_Lexer::MULTILINESTRING_SYMBOL    => 'multilinestring',
+		WP_MySQL_Lexer::POLYGON_SYMBOL            => 'polygon',
+		WP_MySQL_Lexer::MULTIPOLYGON_SYMBOL       => 'multipolygon',
+		WP_MySQL_Lexer::JSON_SYMBOL               => 'json',
+	);
+
+	/**
+	 * Database name.
+	 *
+	 * @var string
+	 */
+	private $db_name;
+
+	/**
+	 * Query callback.
+	 *
+	 * @var callable
+	 */
+	private $query_callback;
+
+	public function __construct( string $db_name, callable $query_callback ) {
+		$this->db_name        = $db_name;
+		$this->query_callback = $query_callback;
+	}
+
+	public function ensure_tables(): void {
+		foreach ( self::CREATE_INFORMATION_SCHEMA_QUERIES as $query ) {
+			$this->query( $query );
+		}
+	}
+
+	public function create_table( WP_Parser_Node $node ): void {
```

Comment:
Just `create_table` suggests this method actually creates a table. How about `store_create_table` or `notify_create_table` or something to that effect?

Reply:
@adamziel Makes sense! I'll revisit the naming as well, there's likely more to improve at the moment.